### DISCUSSION TITLE
build error on latest Next@14

### DISCUSSION THREAD

**Discussion by mydearxym** [chakra-ui/panda#1820] [Q&A]
love the project !

hi team recently i plan to switch from styled-component to panda, i referred configs from [sandbox project](https://github.com/chakra-ui/panda/tree/main/sandbox/next-js-app), and works well on DEV mode:

<img width="665" alt="image" src="https://github.com/chakra-ui/panda/assets/6184465/54e3c8e8-382f-4370-a760-d4767f4b9d3b">

but comes to build, i got type error: 

<img width="615" alt="image" src="https://github.com/chakra-ui/panda/assets/6184465/90590e7c-6f88-42d4-9389-db57861b6232">

and if i add @ts-ignore to it, it will pop out another issue:

<img width="900" alt="image" src="https://github.com/chakra-ui/panda/assets/6184465/bc088b14-c6af-4e00-8f6a-c490c4ef91d5">

env:
```text
Fromework: Next.js@14.0.4
PandaCSS: @pandacss/dev": "^0.21.0"
Typescript: typescript@5.3.3
OS: Mac
Chrome: 119
```

demo repo: 

https://github.com/groupher/panda-test 

i am not sure if it's a bug or i am missing some config options, has anyone encountered a similar situation?

**Comment by segunadebayo** [comment]
This is most likely a type regression. Will look into it shortly
